templates/base.html

```diff
@@ -7,7 +7,8 @@
     <meta name="robots" content="noindex, nofollow">
     <title>Take Your Meds - Phone reminders to help you and yours remember to take your medication</title>
 
-    <link href='//fonts.googleapis.com/css?family=Merriweather:300,400,700' rel='stylesheet' type='text/css' />
+    <link href='http://fonts.googleapis.com/css?family=Merriweather:300,400,700' rel='stylesheet' type='text/css' />
```

Comment:
This will load insecure content over a page that should (hopefully!) be SSL, causing a browser warning/padlock blah.